src/murmur/UnixMurmur.cpp

```diff
@@ -228,7 +228,11 @@ void UnixMurmur::handleSigUsr1() {
 	ssize_t len = ::read(iUsr1Fd[1], &tmp, sizeof(tmp));
 	Q_UNUSED(len);
 
-	qWarning("Received USR1 signal... Ignoring...");
+	if (meta) {
+		qWarning("Meta: trying to reload SSL settings");
+		meta->reloadSSLSettings();
+		qWarning("Meta: done reloading SSL settings");
```

Comment:
Shouldn’t this indicate success or failure? We have a return value with that information.